go.mod

```diff
@@ -128,15 +130,15 @@ require (
 	github.com/vmware/govmomi v0.20.1
 	github.com/xlab/handysort v0.0.0-20150421192137-fb3537ed64a1 // indirect
 	golang.org/x/crypto v0.0.0-20190611184440-5c40567a22f8
-	golang.org/x/lint v0.0.0-20181217174547-8f45f776aaf1
+	golang.org/x/lint v0.0.0-20190301231843-5614ed5bae6f
```

Comment:
https://github.com/golang/lint/branches doesn't have a branch corresponding to the go release branch, so there's no need to stay pinned to `v0.0.0-20181217174547-8f45f776aaf1`. prefer the maximal common version (`v0.0.0-20190301231843-5614ed5bae6f`)